canvas_modules/common-canvas/src/tooltip/tooltip.jsx

```diff
@@ -119,7 +119,8 @@ class ToolTip extends React.Component {
 	// (scrollWidth) value is equal to the minimum width the element would require
 	//  in order to fit all the content in the viewport without using a horizontal scrollbar
 	canDisplayFullText(elem) {
-		if (elem) {
+		const input = elem.getElementsByTagName("input")[0];
```

Comment:
This doesn't seem like the right place for this.  It's too specific to `input` elements and the tooltip component is more generic.  I would probably pass in a new property into the tooltip component to check.  Something like `truncatedElement`.  Then pass the `input` element into that.  It would be optional and fall back to this.props.children if not set.